src/main/java/com/theelixrlabs/healthcare/constants/PatientConstants.java

```diff
@@ -9,13 +9,16 @@ public class PatientConstants {
     public static final String AADHAAR_NUMBER_REGEX = "^[2-9][0-9]{11}";
     public static final String ALPHA_CHARACTERS_ONLY_REGEX = "^[a-zA-Z]+$";
     public static final String CREATE_PATIENT_ENDPOINT = "/addPatient";
```

Comment:
Why it is ALPHA_CHARACTERS_ONLY_REGEX it can be ALPHA_CHARACTERS_REGEX

Reply:
Changed it.